underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -80,15 +80,24 @@ public SwiftUnderFileSystem(AlluxioURI uri, Configuration configuration) {
     String containerName = uri.getHost();
     LOG.debug("Constructor init: {}", containerName);
     AccountConfig config = new AccountConfig();
-    config.setUsername(configuration.get(Constants.SWIFT_USER_KEY));
-    config.setTenantName(configuration.get(Constants.SWIFT_TENANT_KEY));
-    config.setPassword(configuration.get(Constants.SWIFT_API_KEY));
+    if (configuration.containsKey(Constants.SWIFT_API_KEY)) {
```

Comment:
@yupeng9 I don't think we should go so deep into this. Documentation asks to provide password key and it doesn't mention apikey anymore. Users who user older versions, will still able to use apikey. At some point nobody will use apikey, only password.